README.md

```diff
@@ -419,6 +484,20 @@ const raygunClient = new raygun.Client().init({
 
 Including column numbers can enable source mapping if you have minified or transpiled code in your stack traces.
 
+### APM Bridge Setup
+
+Raygun4Node can be used in conjunction with Raygun APM for Node.js.
+
+Follow the instructions on https://raygun.com/platform/apm to set up your application.
+
+Then, call to `setApmBridge()` to connect both packages:
```

Comment:
Add a full example of how this works. Something like:
`
require('raygun-apm/http');
const raygun = require('raygun');
raygun.setApmBridge();
`

Reply:
It is already in `examples/apm-sample/app.js`

Reply:
Ah you mean in the README, got it.

Reply:
But I think the current text is correct. In other parts of the readme, we don't reference to the `require` for raygun client, and also the `require('raygun-apm/http')` is not required to set up the APM bridge. I also link to the official APM setup process.